### PR TITLE
Data transformation

### PR DESCRIPTION
**Pull request by milmazz**
This PR include a few improvements:
- Now it's possible to show the program version via `cell --version` or `cell -v`
- Use `:inet` instead of `:inet_parse` for the `Inet` module
- Change the code style for commands, now in each command you can see the all workflow of the data transformation.

As always, any feedback is more than welcome.

### CODE REVIEW

**Comment by ghitchens** [comment]
Thanks for all your hard work -- I've not been able to review most of this yet, but hope to do so within the day.

File: lib/cell_tool.ex [hunk 1]
```diff
@@ -34,9 +34,12 @@ defmodule CellTool do
       cell: /jrtp/sys/firmware/current -> ok
   """
 
+  @cell_tool_version Mix.Project.config[:version]
+
   @doc "Function that gets call when run as CLI"
   def main(args) do
-    args |> parse_args
+    args
+      |> parse_args
```
Comment (mobileoverlord) [inline lib/cell_tool.ex:42]: I noticed this earlier where here you are using the indentation for the pipe operator. but (continued below)

File: lib/cmd/denormalize.ex [hunk 2]
```diff
@@ -12,17 +12,17 @@ defmodule Cmd.Denormalize do
...
-        IO.write "already provisional\n"
-      x ->
-        IO.write "DENORMALIZATION FAILED (ERROR #{x})\n"
-    end
+    cell.location
+    |> Path.join("/sys/firmware/current")
+    |> HTTPotion.put(~s({"status": "provisional"}), ["Content-Type": "application/json"])
+    |> verify_status()
+    |> response("#{cell.name} -> ")
+    |> IO.write()
```
Comment (mobileoverlord) [inline lib/cmd/denormalize.ex:20]: Here you are not using the indentation for the pipe operator.

**Comment by mobileoverlord** [comment]
@milmazz I think its awesome that you are taking the time to lint this code. I'm just curious about the pipe indentation. I've noticed in `mix phoenix.gen.*` the code it generates is also not indenting the pipe. Is this the going forward style choice?

**Comment by milmazz** [comment]
@mobileoverlord Good catch! Normally I prefer this way:

``` elixir
cell.location
|> Path.join("/sys/firmware/current")
|> HTTPotion.put(~s({"status": "provisional"}), ["Content-Type": "application/json"])
|> verify_status()
|> response("#{cell.name} -> ")
|> IO.write()
```

... and this way when you bind the result of a computation:

``` elixir
file_ext =
  input
  |> Path.extname()
  |> String.downcase()
```

Instead of this:

``` elixir
cell.location
  |> Path.join("/sys/firmware/current")
  |> HTTPotion.put(~s({"status": "provisional"}), ["Content-Type": "application/json"])
  |> verify_status()
  |> response("#{cell.name} -> ")
  |> IO.write()
```

I can fix this indentation problems in another PR if you want.

What do you think? /cc @ghitchens

File: lib/cell_tool.ex (hunk 1 above)
Comment (ghitchens) [inline lib/cell_tool.ex:42]: @milmazz @mobileoverlord took care of it already.   Thanks!

**Pull request merged**